src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/tests/test_lab.py

```diff
@@ -4,137 +4,146 @@
 # --------------------------------------------------------------------------------------------
 
 import os
-from azure.cli.testsdk.vcr_test_base import ResourceGroupVCRTestBase, JMESPathCheck, NoneCheck
 from azure.cli.testsdk import record_only
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
 
-TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
-TEMPLATE = '{}/lab_template.json'.format(TEST_DIR)
-ENV_PARAMTERS = '@{}/docdbenv_paramters.json'.format(TEST_DIR)
+TEST_DIR = os.path.dirname(os.path.realpath(__file__))
+TEMPLATE = os.path.join(TEST_DIR, 'lab_template.json').replace('\\', '\\\\')
+ENV_PARAMETERS = os.path.join('@' + TEST_DIR, 'lab_template.json').replace('\\', '\\\\')
 LAB_NAME = 'cliautomationlab'
 
 
-class LabGalleryVMMgmtScenarioTest(ResourceGroupVCRTestBase):
-
-    def __init__(self, test_method):
-        super(LabGalleryVMMgmtScenarioTest, self).__init__(__file__, test_method,
-                                                           resource_group='cliautomation')
+class LabGalleryVMMgmtScenarioTest(ScenarioTest):
 
     @record_only()
-    def test_lab_gallery_vm_mgmt(self):
-        self.execute()
-
-    def body(self):
-        rg = self.resource_group
-        linux_vm_name = 'ubuntuvm5367'
-        linux_image = 'Ubuntu\ Server\ 16.04\ LTS'
-        windows_vm_name = 'winvm5367'
-        windows_image = 'Windows\ Server\ 2008\ R2\ SP1'
-        image_type = 'gallery'
-        size = 'Standard_DS1_v2'
-        password = 'SecretPassword123'
-
-        self.cmd('group deployment create -g {} --template-file {}'
-                 .format(rg, TEMPLATE),
-                 checks=[JMESPathCheck('properties.provisioningState', 'Succeeded')])
+    @ResourceGroupPreparer(name_prefix='cliautomation')
+    def test_lab_gallery_vm_mgmt(self, resource_group):
+        self.kwargs.update({
+            'resource_group': resource_group,
```

Comment:
The ResourceGroupPreparer automatically populates kwargs with this value under 'rg'.  Please update accordingly.